docs/user/user/metrics/error-reporting.md

```diff
@@ -11,6 +11,7 @@ The following categories of errors are recorded:
 - `invalid_label`: The label on a labeled metric was invalid.
 - `invalid_state`: The metric caught an invalid state while recording.
 - `invalid_overflow`: The metric value to be recorded overflows the metric-specific upper range.
+- `invalid_type`: The metric value is not of the expected type. This error type is only recorded by the Glean JavaScript SDK. This error may only happen in dynamically typed languages.
```

Comment:
ah, here we have the explanation.

Reply:
Yeap, thought I'd add it here so I don't need to repeat myself in all the metrics files (we would forget to do that everytime, probably). With regards to filing a bug to do this in Python and JS, that is up to you and @chutten I guess. Not sure wha are the priorities for the FOG and Python SDKs right now.

Reply:
For it to work in Firefox Desktop's JS we'd need https://bugzilla.mozilla.org/show_bug.cgi?id=1691073